assignments/prototypes.js

```diff
@@ -32,6 +53,19 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+function Humanoid(hAttributes){
+  CharacterStats.call(this, hAttributes);
+  this.team=hAttributes.team;
+  this.weapons=hAttributes.weapons;
+  this.language=hAttributes.language;
+
+}
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+Humanoid.prototype.greet=function(){
+  return '<object name> offers a greeting in <object language>.';
+}
+
```

Comment:
Same thing here, use interpolation and add this.name and this.language to the return statement